internal/api/sms_provider/messagebird.go

```diff
@@ -55,6 +55,15 @@ func NewMessagebirdProvider(config conf.MessagebirdProviderConfiguration) (SmsPr
 	}, nil
 }
 
+func (t *MessagebirdProvider) SendMessage(phone string, message string, messageType string) error {
+	switch messageType {
+	case SMSProvider:
+		return t.SendSms(phone, message)
+	default:
+		return nil
+	}
+}
```

Comment:
I guess if the `messageType` is not correct here, it should return an error because it's a programmer problem that the code reached this far in. Also can we call it `channel` or `messageType` (I prefer channel) consistently?